lib/cylc/remote.py

```diff
@@ -69,7 +71,7 @@ def __init__(self):
         self.is_remote = (
             is_remote_user(self.owner) or is_remote_host(self.host))
 
-    def execute(self, force_required=False, env=None, path=None):
+    def execute(self, force_required=False, env={}, path=None, dry_run=False):
```

Comment:
the `env={}` can cause nasty problems if this function is ever called more than once - the `env` variable
becomes effectively an attribute of the `remrun` class, rather than an independent variable:
http://stackoverflow.com/questions/9887180/the-value-of-an-empty-list-in-function-parameter-example-here